src_wires_pub/wires.F90

```diff
@@ -6975,7 +6975,7 @@ subroutine wiresconstantes(fieldtotl,dummy,G2,sgg)
           
           logical, intent(in) :: fieldtotl
           type (CurrentSegments), pointer  ::  dummy
-!!!ojooooo ����110517 acumulo en %lind toda la autoinduccion para que los calculos de capacidad la tengan en cuenta completa    
+!!!ojooooo ����110517 acumulo en %lind toda la autoinduccion para que los calculos de capacidad la tengan en cuenta completa    

```

Comment:
Seems that unicode characters are problematic for some editor. Please change to plain ASCII letters.